app-misc/elasticsearch/Manifest

```diff
@@ -1,3 +1,2 @@
-DIST elasticsearch-1.7.6.tar.gz 28488767 BLAKE2B 0dc4fb10355e2139ee3a7e7e65366e8aa533b9714bf0f00d2bd8f4ef0669ba9f8f10dc5255c8c7a62abc0362e08ab5fda3c8b6b6f2bacfe5cce7255973293162 SHA512 2746f09b66919912e785c8d25c50ae74dbb38a41d99d9c926550786f9c2d7953d1ae1a54a32dae59a7269e5fbfa74dfbc91c3d412e415efd01c391ef7d329ba7
```

Comment:
did you talk to tony about this? newer sugarcrm versions support 5.6 but i don't use it so i don't know how much of a headache upgrading that would be.
(it really was added even *after* its EOL so ehhhh...)

Reply:
Well I told him to remove the package as it's EOL. I did not receive any reply, but it does not change the fact this version is obsolete and not maintained. If anyone wishes to use it, simply put it in your overlay and continue using it.